src/api.c

```diff
@@ -2144,19 +2144,21 @@ int cg_mkdir_p(const char *path)
 			case EPERM:
 				ret = ECGROUPNOTOWNER;
 				goto done;
-			default:
-				/* Check if path exists */
-				real_path[i] = '\0';
-				stat_ret = stat(real_path, &st);
-				real_path[i] = pos;
-				if (stat_ret == 0) {
-					ret = 0;	/* Path exists */
-					break;
-				}
+			case EROFS:
```

Comment:
The  reason for explicitly checking for `EROFS` errno is  captured in the commit 74b3eb1db015